src/main/scala/io/iohk/praos/App.scala

```diff
@@ -72,7 +74,7 @@ object App extends Logger {
           )
           val newBlock: Block = blockFactory.makeBlock(
             slotInEpoch.slotNumber,
-            isLeaderProof.get,
+            isLeaderProof.get._2,
```

Comment:
Minor detail: Instead of doing a:
```
if(x.isDefined)
   ... x.get ...
else
   ...
```
maybe it would be better to do something like:
```
x match {
   case Some(xDefined) => ...
   case None => ...
   ...
```

Reply:
:100: